hack/test-templates.sh

```diff
@@ -389,6 +396,10 @@ if [[ -n ${CHECKS["snapshot-offline"]} ]]; then
 	limactl start "$NAME"
 fi
 
+if [[ $NAME == "fedora" && "$(limactl ls --json "$NAME" | jq -r .vmType)" == "vz" ]]; then
```

Comment:
This should check `limactl shell "$NAME" getenforce`, not the name

Reply:
getenforce would be available on fedro based distros, hence the name is used. For other distros, we need to handle the command being not found error.